helpers/helpers-spec_service.bal

```diff
@@ -213,4 +215,21 @@ service / on main_endpoint {
             }
         };
     }
+
+    resource function get countries() returns Country[]|http:InternalServerError {
+
+        // Read the CSV content as a string
+        string|error csvContent = io:fileReadString("resources/countries.csv");
+        if csvContent is error {
+            log:printError("Error reading CSV file", err = csvContent.message());
+            return <http:InternalServerError>{body: csvContent.message()};
```

Comment:
Are these error details useful to the end user?

Reply:
+1 , i think we better remove that much of details.